src/Entity/Message.php

```diff
@@ -302,21 +299,18 @@ public function getText($langcode = Language::LANGCODE_NOT_SPECIFIED, $options =
           $value = call_user_func_array($value['callback'], $value['arguments']);
         }
 
+        // @todo Replace some, or all, of this with core methods.
```

Comment:
I think this is pretty important, so we should tackle this here - what were you think of doing?

We should probably also mention `FormattableMarkup::placeholderFormat` here, which this function is now mimicking.